assets/src/edit-story/components/panels/textStyle/color.js

```diff
@@ -74,6 +89,31 @@ function ColorControls({ selectedElements, onSetProperties }) {
           label={__('Background color', 'web-stories')}
         />
       </Row>
+      {backgroundColor && (
+        <Row>
+          <Space flex="1" />
+          <ToggleButton
```

Comment:
I would expect to be able to toggle these with arrow keys too, when focused. Should be quite simple with `useKeyDownEffect`.

Reply:
Sure! I will take a look at examples using `useKeyDownEffect ` to achieve this. Just to clarify, using <kbd><</kbd> or <kbd>></kbd> you want to be able to switch between both, right?

Reply:
I wouldn't expect it to cycle. Just pressing <kbd>left</kbd> moves to the next button to the left (and selects it). Clicking <kbd>left</kbd> when already at the left-most button does nothing.